App/src/pages/dossier/DossierReview.tsx

```diff
@@ -241,6 +260,73 @@ export default function DossierReview() {
         );
     }
 
+    const handleForwardDossier = () => {
+        forwardDossier(dossierId)
+            .then(() => {
+                showToast(toast, "Success!", "Dossier successfully forwarded.", "success");
+            })
+            .catch(() => {
+                showToast(toast, "Error!", "One or more validation errors occurred", "error");
+            });
+    };
+
+    const handleReturnDossier = () => {
+        returnDossier(dossierId)
+            .then(() => {
+                showToast(toast, "Success!", "Dossier successfully returned.", "success");
+            })
+            .catch(() => {
+                showToast(toast, "Error!", "One or more validation errors occurred", "error");
+            });
+    };
+
+    const handleRejectDossier = () => {
+        rejectDossier(dossierId)
+            .then(() => {
+                showToast(toast, "Success!", "Dossier successfully rejected.", "success");
+            })
+            .catch(() => {
+                showToast(toast, "Error!", "One or more validation errors occurred", "error");
+            });
+    };
+
+    const handleChangeMessage = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+        if (e.currentTarget.value.length === 0) setMessageError(true);
+        else setMessageError(false);
+        setMessage(e.currentTarget.value);
+    };
+
+    const handleSubmitRequest = () => {
+        setFormSubmitted(true);
+        if (messageError) return;
+        else {
+            const dossierForReviewDTO = {
+                message: message,
+                groupId: currentGroup.groupId,
+            };
+
+            reviewDossier(dossierId, dossierForReviewDTO)
+                .then(() => {
+                    showToast(toast, "Success!", "Message successfully sent.", "success");
```

Comment:
I think after submitting the message and showing the toast, you should clear the input box, rather than keeping the old message still written there.

Reply:
Fixed.